src/pykeen/models/unimodal/boxe_kg.py

```diff
@@ -0,0 +1,114 @@
+from typing import Any, ClassVar, Mapping, Optional
+
+from torch.nn.init import uniform_
+
+from ...constants import DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE
+from ...models import ERModel
+from ...nn.emb import EmbeddingSpecification
+from ...nn.init import uniform_norm_
+from ...nn.modules import BoxEKGInteraction
+from ...typing import Hint, Initializer
+
+__all__ = [
+    "BoxEKG",
+]
+
+
+class BoxEKG(ERModel):
+    r"""An implementation of BoxE.
+
+    ---
+    citation:
+        author: Abboud
+        year: 2020
+        link: https://arxiv.org/abs/2007.06267
+    """
+
+    #: The default strategy for optimizing the model's hyper-parameters
+    hpo_default: ClassVar[Mapping[str, Any]] = dict(
+        embedding_dim=DEFAULT_EMBEDDING_HPO_EMBEDDING_DIM_RANGE,
+        p=dict(type=int, low=1, high=2),
+    )
+
+    def __init__(
+        self,
+        *,
+        embedding_dim: int = 200,
+        tanh_map: bool = True,
+        norm_order: int = 2,
+        entity_initializer: Hint[Initializer] = uniform_norm_,
+        entity_initializer_kwargs: Optional[Mapping[str, Any]] = None,
+        relation_initializer: Hint[Initializer] = uniform_norm_,  # Has to be scaled as well
+        relation_initializer_kwargs: Optional[Mapping[str, Any]] = None,
+        relation_size_initializer: Hint[Initializer] = uniform_,  # Has to be scaled as well
+        relation_size_initializer_kwargs: Optional[Mapping[str, Any]] = None,
+        **kwargs,
+    ) -> None:
+        r"""Initialize BoxE-KG
+
+        :param embedding_dim: The entity embedding dimension $d$. Defaults to 200. Is usually $d \in [50, 300]$.
+        :param tanh_map: Whether to use tanh mapping after BoxE computation. Default - True
```

Comment:
can you give insight here as to why you would want to use or not use this mapping?

Reply:
Sure!